torch/_inductor/__init__.py

```diff
@@ -332,6 +332,21 @@ def list_mode_options(
 
     mode_options: dict[str, dict[str, bool]] = {
         "default": {},
+        # lightweight backend
```

Comment:
@BoyuanFeng  just a question about how the config works.  in autoparallel, we directly call `compile_fx_inner` from `aot_export_joint_...`'s compile_fn.    Can I directly use this 'mode' for light somewhere?  or do i just need to manually set these same set of options in the inductor.config module?